python/src/GaussProductExperiment_doc.i.in

```diff
@@ -2,25 +2,87 @@
+
+The Gauss tensor product experiment uses Gauss points in each direction. For each marginal, the algorithm computes the family of orthogonal polynomials depending on the marginal distribution using the :class:`~openturns.StandardDistributionPolynomialFactory` class. The input distribution must have an independent copula.  
+
+The :meth:`generate` method always returns the same sample, i.e. this is a 
+deterministic DOE.
+
+Let :math:`\mathcal{X} \subset \mathbb{R}^{n_x}` be the integration domain and let :math:`g : \mathcal{X} \rightarrow \mathbb{R}^{n_y}` be an integrable function. Let :math:`f : \mathcal{X} \rightarrow \mathbb{R}` be a probability density function. The Gauss tensor product experiment produces an approximation of the integral:
+
+.. math::
+
+    \int_{\mathcal{X}} g(\boldsymbol{x}) f(\boldsymbol{x}) d\boldsymbol{x} \approx \sum_{i = 1}^{s_t} w_i g\left(\boldsymbol{x}_i\right)
+
+where :math:`s_t \in \mathbb{N}` is the size of the tensor product design of experiments, :math:`w_1, ..., w_{s_t} \in \mathbb{R}` are the weights and :math:`\boldsymbol{x}_1, ..., \boldsymbol{x}_{s_t} \in \mathbb{R}^{n_x}` are the nodes. 
+
+Let :math:`\boldsymbol{k} = \left(k_1, ..., k_{n_x}\right) \in (\mathbb{N}^\star)^{n_x}`  be a multi-index where :math:`\mathbb{N}^\star = \{1, 2, ..., \}` is the set of natural numbers without zero. For each marginal index :math:`j = 1, ..., n_x`, let :math:`s_j \in \mathbb{N}` be its size, representing the number of nodes in the marginal quadrature. Let :math:`\left\{ Q_{s_j}^{(1)} \right\}_{j = 1, ..., n_x}` be the collection of marginal experiments where the exponent :math:`(1)` denotes univariate quadrature rule. 
+
+The size of the tensor product experiment is equal to the product of the marginal sizes:
+
+.. math::
+
+    s_t = \prod_{j = 1}^{n_x} s_j.
+
+The tensor product quadrature is ([gerstner1998]_ page 214):
+
+.. math::
+
+    Q^{(n_x)}_{s_t} = Q_{s_1}^{(1)} \otimes \cdots \otimes Q_{s_{n_x}}^{(1)}
+
+where:
+
+.. math::
+
+    Q_{s_1}^{(1)} \otimes \cdots \otimes Q_{s_{n_x}}^{(1)} (g) = \sum_{k_1 = 1}^{s_1} \cdots \sum_{k_{n_x} = 1}^{s_{n_x}} w_{s_1, k_1} \cdots w_{s_{n_x}, k_{n_x}}  g\left(x_{s_1, k_1}, \ldots, x_{s_{n_x}, k_{n_x}}\right).
+
+For any multi-index :math:`\boldsymbol{k} = (k_1, ..., k_{n_x}) \in (\mathbb{N}^\star)^{n_x}`, we write 
+
+.. math::
+
+    \boldsymbol{k} \leq \left(s_1, ..., s_{n_x}\right)
+
+if and only if :math:`k_j \leq s_j` for :math:`j = 1, ..., n_x`. This means that each component of the multi-index is lower than its corresponding marginal size.
+The set of multi-indices such that :math:`\boldsymbol{k} \leq \left(s_1, ..., s_{n_x}\right)` is produced using all the possible combinations of the indices with the :class:`~openturns.Tuples` class.
```

Comment:
I find this sentence unclear. By definition, this set contains all possible combinations of the indices that fit the constraint. What are you trying to say?

Reply:
I want to write the word "combination" in plain english, and point to the `Tuples` class.

Reply:
I consider this as "resolved", but feel free to reopen it if required.